File: Exercicios_guanaraba/Com_Dart/bin/ex006_Dobro_Triplo_Raiz.dart
//Crie u algoritmo que leia um número e mostre o seu dobro, triplo e raizquadrada.
import 'dart:io';

import 'dart:math';

void main (){

print ('Insira um numero: ');
int numero = int.parse(stdin.readLineSync());

print ('O brodro é: ' + (numero*2).toString());
print ('O brodro é: ' + (numero*3).toString());
print ('A raiz quadrada é: ' + sqrt(numero).toString());



}

File: Exercicios_guanaraba/Com_Ruby/exercicios/ex006_Dobro_Triplo_Raiz.rb
#Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raizquadrada.

puts 'Insira um numero: '
numero = gets.chomp.to_i

puts "O dobro dele é: #{numero*2}"
puts "O triplo dele é: #{numero*3}"

if numero >= 0
    puts "A raiz quadrada de #{numero} é #{numero**0.5}"
    puts "Usando metodo sqrt a raiz de #{numero} é #{Math.sqrt(numero)}"
else
    puts'Não pode ser um numero menor ou igual a 0'
end